release/create_release_pull_request.py

```diff
@@ -58,7 +58,7 @@ def validate_version(version):
 
 def generate_changelog():
     print("Generating changelog...")
-    subprocess.run(["./generate_release_changelog.sh"], check=True, cwd=SCRIPTS_DIR)
+    subprocess.run(["./generate_release_changelog.sh", "-no-prompt"], check=True, cwd=SCRIPTS_DIR)
```

Comment:
Should have one more dash prefixed to match arg in `generate_release_changelog.sh`